groups/instance/lb.tf

```diff
@@ -0,0 +1,114 @@
+resource "aws_lb" "artifactory" {
+  name               = "${var.environment}-${var.service}-lb"
+  internal           = true
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.alb_security_group.id]
+  subnets            = local.automation_subnet_cidrs
```

Comment:
This should be a list of subnet IDs, rather than CIDR blocks, so would be worth verifying that this data structure contains the correct information. If it does indeed hold subnet IDs, then I would recommend renaming the var to better reflect it's contents.